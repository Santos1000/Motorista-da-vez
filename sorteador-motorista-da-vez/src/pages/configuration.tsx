import { Footer } from "../components/footer/footer";
import { Form } from "../components/Form";
import { List } from "../components/listParticipant/participantList";

export function Configuration() {
  return (
    <>
      <Form/>
      <List/>
      <Footer/>
    </>
  )
}
