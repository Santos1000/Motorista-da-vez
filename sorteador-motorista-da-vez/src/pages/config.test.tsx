import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil"
import { Configuration } from "./configuration"

const mockNavigation = jest.fn()

jest.mock('react-router-dom', () => {
  return {
    useUserList: () => mockNavigation
  }
})

describe('configuration page', () => {
  test('render configuration page', () => {
    const { container } = render(
      <RecoilRoot>
        <Configuration/>
      </RecoilRoot>
    )

    expect(container).toMatchSnapshot()
  })
})
