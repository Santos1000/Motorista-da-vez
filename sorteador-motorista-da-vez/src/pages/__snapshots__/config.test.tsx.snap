// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration page render configuration page 1`] = `
<div>
  <form>
    <input
      placeholder="Enter participant names"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Start game
    </button>
  </footer>
</div>
`;
